pilot/pkg/serviceregistry/kube/controller/multicluster.go

```diff
@@ -219,40 +218,39 @@ func (m *Multicluster) ClusterAdded(cluster *multicluster.Cluster, clusterStopCh
 					// recreate it again.
 					client.RunAndWait(clusterStopCh)
 					nc.Run(leaderStop)
-				}).Run(clusterStopCh)
+				})
+			// Set up injection webhook patching for remote clusters we are controlling.
+			// The local cluster has this patching set up elsewhere. We may eventually want to move it here.
```

Comment:
@stevenctl with this code added here, do you think we can remove it from the init code?
https://github.com/istio/istio/blob/ac5ae313be27941e6a2eb59f24cf68db2b9a5813/pilot/pkg/bootstrap/sidecarinjector.go#L89

Reply:
I tried removing this but it didn't work. Lots of problems with revision tests.